src/command_modules/azure-cli-servicebus/azure/cli/command_modules/servicebus/_exception_handler.py

```diff
@@ -0,0 +1,37 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+
+def response_exception_handler(ex):
+    from azure.mgmt.servicebus.models import ErrorResponseException
+    from knack.util import CLIError
+    import json
+
+    if isinstance(ex, ErrorResponseException) and ex.response.json():
```

Comment:
The logic you had previously was okay. You just needed to handle the ValueError that is thrown when response.json() fails.
ex.response.json() will throw an exception if it fails, instead of returning a value that evaluates to false. 
You need to add a try / except for this case, instead of moving the call to response.json() into the if statement.

Please see [here](http://docs.python-requests.org/en/master/api/#requests.Response.json). It shows you what response.json() throws.

Reply:
Sure, will implement.